src/leaderboard.js

```diff
@@ -0,0 +1,90 @@
+import axios from 'axios';
+
+let gameID;
+const body = document.querySelector('body');
+
+async function createGame(name) {
+  const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+  try {
+    const response = await axios.post(baseURL, { name });
```

Comment:
-  [ ] [REQUIRED] This code does not use async and await keywords with the fetch method. Instead, it uses the axios library for making HTTP requests. Kindly ensure you stick to the requirements of the project by using async and await keywords with the fetch method.